preprocess.py

```diff
@@ -40,18 +41,30 @@ def get_parser():
     parser.add_argument('--only-source', action='store_true', help='Only process the source language')
     parser.add_argument('--padding-factor', metavar='N', default=8, type=int,
                         help='Pad dictionary size to be multiple of N')
+    parser.add_argument('--tokenize_number', metavar='N', default=0, type=int, choices=range(2),
+                        help="Which tokenization algorithm to use. 0 is tokenize_line, 1 is nltk's word tokenize. Default is 0.")
```

Comment:
it is better to use string arguments here with a choices=['default', 'nltk'], here and then store them in a dict instead of array. also can you please describe what the difference between the two tokenization strategies are, and add a summary of that description into the help string?